### DISCUSSION TITLE
Pedestal pressure scaling for ST

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#3489] [Ideas]
In GitLab by @skahn on Mar 24, 2021, 13:05

## Description

Samuli Saalerma (EPED author) proposed a new scaling for STEP to capture the pedestal pressure triangularity dependence. It may be very nice to use it to define the pedestal temperature !

Here are the plots data points he gently provided to me:

![image](https://git.ccfe.ac.uk/process/process/uploads/69d9207c7ab2c13628cc43b3c591ec8d/image.png)

triangularity:0.2500    0.3000    0.4000    0.5000    0.6000    0.7000
blue criterion:  22.4678   25.9126   38.4566   54.3749   71.6242   98.6992
red criterion: 25.6060   27.6459   37.8634   53.0704   81.3390  110.8707

His message was that both criterion were equivalent for ST at the moment.

## Proposal

Implement this scaling and use it :)

Best regards     
Sebastien Kahn

**Comment by mkovari** [comment]
Please remember that trying to implement pedestal scalings has proved to be a can of worms.
